File: uri/Organizar/1051 - Imposto de renda.cpp
#include <iostream>
#include<iomanip>

using namespace std;

int main() {

    float salario;
    cin>>salario;
    if(salario >= 0 && salario<=2000){
        cout<<"Isento\n";
    }else{
        if(salario>2000 && salario <=3000){
            cout<<fixed<<setprecision(2)<<"R$ "<<((salario - 2000)*0.08)<<"\n";
        }else{
            if(salario>3000 && salario<=4500){
                cout<<fixed<<setprecision(2)<<"R$ "<<(80 + (salario-3000)*0.18)<<"\n";
            }else{
                cout<<fixed<<setprecision(2)<<"R$ "<<(350 + (salario-4500)*0.28)<<"\n";
            }
        }
    }
    return 0;
}

File: uri/Organizar/1051 - Imposto de renda.py
salario = float(input())
if salario >= 0.0 and salario <= 2000.0 :
    print 'Isento'
elif salario > 2000.0 and salario <=3000.0:
    print 'R$',round((salario - 2000)*0.08,2)
elif salario>3000 and salario<=4500:
    print 'R$',round(80 + (salario-3000)*0.18,2)
else :
    print 'R$',round(350 + (salario-4500)*0.28,2)
